roachtest: schemachange/invertedindex failed

**Note:** This build has runtime assertions enabled. If the same failure was hit in a run without assertions enabled, there should be a similar failure without this message. If there isn&#39;t one, then this failure is likely due to an assertion violation or (assertion) timeout.

roachtest.schemachange/invertedindex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18531849?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/18531849?buildTab=artifacts#/schemachange/invertedindex) on release-25.1 @ [f9c2a2f417a567a1dc6e7b23b41b8759cf62d077](https://github.com/cockroachdb/cockroach/commits/f9c2a2f417a567a1dc6e7b23b41b8759cf62d077):


```
(cluster.go:2478).Run: full command output in run_135140.662279828_n5_workload-run-json-re.log: COMMAND_PROBLEM: exit status 1
(monitor.go:149).Wait: monitor failure: monitor user task failed: t.Fatal() was called
test artifacts and logs in: /artifacts/schemachange/invertedindex/run_1
```

Parameters:
 - <code>arch=amd64</code>
 - <code>cloud=gce</code>
 - <code>coverageBuild=false</code>
 - <code>cpu=4</code>
 - <code>encrypted=false</code>
 - <code>metamorphicLeases=expiration</code>
 - <code>runtimeAssertionsBuild=true</code>
 - <code>ssd=0</code>
<details><summary>Help</summary>
<p>


See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)



See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)



See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-18531849/schemachange-invertedindex/1737121885918/1737123108999)

</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #138891 roachtest: schemachange/invertedindex failed [split failed while applying backpressure; error happened before the schema change was executed] [B-runtime-assertions-enabled C-test-failure O-roachtest O-robot T-sql-foundations branch-master]
</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*schemachange/invertedindex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-46599

Workload failed with:

```
E250117 14:09:23.916535 1 workload/cli/run.go:599  [-] 4  workload run error: pq: split failed while applying backpressure to Put [/Table/106/1/136264000/0], Put [/Table/106/1/136264001/0], Put [/Table/106/1/136264002/0], Put [/Table/106/1/136264003/0], Put [/Table/106/1/136264004/0], Put [/Table/106/1/136264005/0], Put [/Table/106/1/136264006/0], Put [/Table/106/1/136264007/0], Put [/Table/106/1/136264008/0], Put [/Table/106/1/136264009/0], Put [/Table/106/1/136264010/0], Put [/Table/106/1/136264011/0], Put [/Table/106/1/136264012/0], Put [/Table/106/1/136264013/0], Put [/Table/106/1/136264014/0], Put [/Table/106/1/136264015/0], Put [/Table/106/1/136264016/0], Put [/Table/106/1/136264017/0], Put [/Table/106/1/136264018/0], Put [/Table/106/1/136264019/0],... 976 skipped ..., Put [/Table/106/1/136264996/0], Put [/Table/106/1/136264997/0], Put [/Table/106/1/136264998/0], Put [/Table/106/1/136264999/0], EndTxn(parallel commit) [/Table/106/1/136264000/0], [txn: d657bdfb], [can-forward-ts] on range r94:/{Table/106/1/127030358-Max} [(n3,s3):1, (n2,s2):2, (n4,s4):3, next=4, gen=66]: operation "split queue process replica 94" timed out after 1m5.577s (given timeout 1m0s): failed to re-compute MVCCStats pre split: result is ambiguous: after 46.48s of attempting command: context deadline exceeded
```

Sending to KV

--- a/pkg/cmd/roachtest/tests/inverted_index.go
+++ b/pkg/cmd/roachtest/tests/inverted_index.go
@@ -55,7 +55,7 @@ func runSchemaChangeInvertedIndex(ctx context.Context, t test.Test, c cluster.Cl
 	m := c.NewMonitor(ctx, c.CRDBNodes())
 
 	cmdWrite := fmt.Sprintf(
-		"./workload run json --read-percent=0 --duration %s {pgurl%s} --batch 1000 --sequential",
+		"./workload run json --read-percent=0 --duration %s {pgurl%s} --batch 200 --sequential",
 		initialDataDuration.String(), c.CRDBNodes(),
 	)
 	m.Go(func(ctx context.Context) error {

